Exception occurred
Traceback (most recent call last):
  File "server.py", line 185, in get
    if not data["version"] or not data["linkAndroid"] or not data["linkIos"]:
KeyError: 'version'
